eq-author/src/App/qcodes/QCodesTable/index.js

```diff
@@ -187,7 +193,9 @@ const Row = memo((props) => {
 
   const handleBlurOptionValue = useCallback(
     (value) => {
-      updateValue(mutationVariables({ id, value }));
+      const trimmedValue = value.trim().replace(/\s+/g, " ");
```

Comment:
Please change this to `const trimmedValue = value?.trim().replace(/\s+/g, " ");`